graphql/schema/directives.graphql

```diff
@@ -18,6 +25,13 @@ requireDistroAccess is used to restrict view, edit, admin, and create access for
 """
 directive @requireDistroAccess(access: DistroSettingsAccess!) on ARGUMENT_DEFINITION | INPUT_FIELD_DEFINITION
 
+enum DistroSettingsAccess {
+  ADMIN
+  CREATE
```

Comment:
i think that makes sense, since this is how we currently check for create permission! I will make some tickets for the follow-up changes soon!

<img width="442" alt="Screenshot 2024-03-21 at 10 37 43 AM" src="https://github.com/evergreen-ci/evergreen/assets/47064971/1cf4315f-0a01-4a31-b931-2c896d4173da">